### PR TITLE
Add 1.28 Release Team

### PR DESCRIPTION
**Pull request by gracenng**
Add [1.28 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.28/release-team.md) to respective groups 

cc @leonardpahlke @saschagrunert

### CODE REVIEW

**Comment by leonardpahlke** [comment]
@gracenng previously the file listed emails and now with your PR Github usernames. Has anything changed in the definition of the groups? Or am I missing something?

**Comment by gracenng** [comment]
woops this should be email not github id's brb

File: groups/sig-release/groups.yaml [hunk 1]
```diff
@@ -67,12 +67,12 @@ groups:
       - joseph.r.sandoval@gmail.com
       - pal.nabarun95@gmail.com
       - cicih@google.com
-      - xandergrzyw@gmail.com # 1.27 Release Lead
-      - kat.cosgrove@gmail.com # 1.27 Release Lead Shadow
-      - priyankasaggu11929@gmail.com # 1.27 Release Lead Shadow
-      - hebaelayoty@gmail.com # 1.27 Release Lead Shadow
-      - salahi.hossein@gmail.com # 1.27 Release Lead Shadow
-      - james.laverack@jetstack.io # 1.27 Emeritus Advisor
+      - gracenng # 1.28 Release Team Lead
```
Comment (leonardpahlke) [inline groups/sig-release/groups.yaml:70]:
```suggestion
      - nng.grace@gmail.com # 1.28 Release Team Lead
```

**Comment by leonardpahlke** [comment]
> brb

😂

**Pull request closed without merging**